lib/sinatra/shopify-sinatra-app.rb

```diff
@@ -189,7 +190,9 @@ def self.registered(app)
         provider :shopify,
                  app.settings.api_key,
                  app.settings.shared_secret,
-
+                 #Fixes for new Shopify callback issues
+                 redirect_uri: app.settings.hostname + "/auth/shopify/callback",
+                 callback_url: app.settings.hostname + "/auth/shopify/callback",
```

Comment:
Hi Kevin,

Fixed as requested. Only thing to make sure of is adding the HOSTNAME
heroku variable to the .env file

On Thu, Nov 26, 2015 at 8:28 PM, Kevin Hughes <notifications@github.com>
wrote:

> In lib/sinatra/shopify-sinatra-app.rb
> <https://github.com/kevinhughes27/shopify-sinatra-app/pull/10#discussion_r46005539>
> :
>
> > @@ -189,7 +190,9 @@ def self.registered(app)
> >          provider :shopify,
> >                   app.settings.api_key,
> >                   app.settings.shared_secret,
> > -
> > +                 #Fixes for new Shopify callback issues
> > +                 redirect_uri: app.settings.hostname + "/auth/shopify/callback",
> > +                 callback_url: app.settings.hostname + "/auth/shopify/callback",
>
> This is what we have in ShopifyApp 'omniauth-shopify-oauth2', '~> 1.1.11'
> https://github.com/Shopify/shopify_app/blob/master/shopify_app.gemspec#L14
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kevinhughes27/shopify-sinatra-app/pull/10/files#r46005539>
> .
>



-- 
Toby Green